packages/features/eventtypes/components/EventTypeDescription.tsx

```diff
@@ -40,6 +42,31 @@ export const EventTypeDescription = ({
 
   const paymentAppData = getPaymentAppData(eventType);
 
+  const showManagedBadge = eventType.metadata?.managedEventConfig && !isPublic;
+  const numOfManagedByAdmins = eventType.managedBy?.admins.length || 0;
+  const showManagedByToolTip = showManagedBadge && numOfManagedByAdmins > 0;
+  let managedByToolTipContent = "";
+  let managedByToolTipContentWithLink;
+
+  if (showManagedByToolTip) {
```

Comment:
As we don't need the `showManagedByTooltip` so instead use the variable that store the admins length and check

Reply:
done